src/formats/airtable-api/formula-converter.ts

```diff
@@ -0,0 +1,786 @@
+		
+		if (inString) {
+			result += char;
+			if (char === stringChar && prevChar !== '\\') {
+				inString = false;
+			}
+		}
+		else {
+			if (char === '"' || char === '\'') {
+				inString = true;
+				stringChar = char;
+				result += char;
+			}
+			else if (char === '&') {
+				// Replace & with +
+				result += '+';
+			}
+			else {
+				result += char;
+			}
+		}
+	}
+	
+	return result;
+}
+
+/**
+ * Convert Airtable's = operator to == for equality comparison
+ * Need to avoid replacing:
+ * - = inside strings
+ * - != (not equal)
+ * - >= (greater than or equal)
+ * - <= (less than or equal)
+ */
+function convertEqualityOperator(formula: string): string {
+	let result = '';
+	let inString = false;
+	let stringChar = '';
+	
+	for (let i = 0; i < formula.length; i++) {
+		const char = formula[i];
+		const prevChar = i > 0 ? formula[i - 1] : '';
+		const nextChar = i < formula.length - 1 ? formula[i + 1] : '';
+		
+		if (inString) {
+			result += char;
+			if (char === stringChar && prevChar !== '\\') {
+				inString = false;
+			}
+		}
+		else {
+			if (char === '"' || char === '\'') {
+				inString = true;
+				stringChar = char;
+				result += char;
+			}
+			else if (char === '=') {
+				// Check if it's part of !=, >=, <=, or already ==
+				if (prevChar === '!' || prevChar === '>' || prevChar === '<' || prevChar === '=') {
+					// Part of !=, >=, <=, or ==, keep as is
+					result += char;
+				}
+				else if (nextChar === '=') {
+					// Already ==, keep as is
+					result += char;
+				}
+				else {
+					// Single = for equality, convert to ==
+					result += '==';
+				}
+			}
+			else {
+				result += char;
+			}
+		}
+	}
+	
+	return result;
+}
+
+/**
+ * Parse comma-separated arguments
+ */
+function parseArguments(argsStr: string): string[] {
+	if (!argsStr.trim()) {
+		return [];
+	}
+	
+	const args: string[] = [];
+	let current = '';
+	let depth = 0;
+	let inString = false;
+	let stringChar = '';
+	
+	for (let i = 0; i < argsStr.length; i++) {
+		const char = argsStr[i];
+		
+		if (inString) {
+			current += char;
+			if (char === stringChar && argsStr[i - 1] !== '\\') {
+				inString = false;
+			}
+		}
+		else {
+			if (char === '"' || char === '\'') {
+				inString = true;
+				stringChar = char;
+				current += char;
+			}
+			else if (char === '(' || char === '[') {
+				depth++;
+				current += char;
+			}
+			else if (char === ')' || char === ']') {
+				depth--;
+				current += char;
+			}
+			else if (char === ',' && depth === 0) {
+				args.push(current.trim());
+				current = '';
+			}
+			else {
+				current += char;
+			}
+		}
+	}
+	
+	if (current.trim()) {
+		args.push(current.trim());
+	}
+	
+	return args;
+}
+
+/**
+ * Find the matching closing parenthesis for an opening parenthesis
+ */
+function findMatchingParen(str: string, openPos: number): number {
+	let depth = 1;
+	let inString = false;
+	let stringChar = '';
+	
+	for (let i = openPos + 1; i < str.length; i++) {
```

Comment:
Not strictly necessary, but if you wanted you might be able to abstract some of this functionality into a helper function which walks along a string, accumulating characters and performing an action unless in quotes.